tests/docker/Dockerfile

```diff
@@ -37,21 +37,13 @@ RUN apt-get update && \
 	firefox xvfb \
 	imagemagick \
 	x11vnc google-chrome-stable cloc \
-	cmake nbtscan python-gammu
```

Comment:
Are there any other send-sms-via-actual-cellphone libraries available?

Reply:
It turns out the package is now appropriately known as **python3-gammu** on Ubuntu. So it's still alive and kicking. We didn't notice as NAV switched to Python 3 because *there are no tests for gammu*.

At least we know it's still available if we do write tests for it.